lib/kube/proxy/moderated_sessions_test.go

```diff
@@ -607,6 +607,9 @@ func TestInteractiveSessionsNoAuth(t *testing.T) {
 	// force the auth client to return an error when trying to create a session.
 	close(testCtx.closeSessionTrackers)
 
+	require.Eventually(t, func() bool {
+		return testCtx.lockWatcher.IsStale()
+	}, 10*time.Second, time.Millisecond*100, "lock watcher should be stale")
```

Comment:
10 seconds is a pretty long time, how much time do we expect this to actually take?

Reply:
It should be fast since the watch() function returns when StaleC closes

Reply:
done